articles/server-platforms/rails/09-customizing-lock.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Customizing Lock
+description: This tutorial will show you how to customize lock widget.
+---
+
+## Ruby On Rails - Customizing Lock
+Using Lock widget is great, but eventually you will want to customize the UI. There are several options to do that.
```

Comment:
From a marketing perspective, the phrasing here could be improved:
"In addition to a carefully-considered default user interface, Lock offers UI customization functionality."